src/chrome/cdtpDebuggee/registries/cdtpScriptsRegistry.ts

```diff
@@ -110,7 +113,10 @@ class CDTPCurrentGeneration {
     }
 
     public getScriptByCdtpId(runtimeScriptCrdpId: string): Promise<IScript> {
-        return this._cdtpIdByScript.get(runtimeScriptCrdpId);
+        // When we attach to a page, we may get Runtime.consoleAPICalled messages with a reference to a ScriptId before we get
+        // the corresponding Debugger.ScriptParsed event. Wait for some time before failing this method, to give time for the message to be parsed
```

Comment:
That's unfortunate. I wonder how chrome devtools handles this.

Reply:
Answered offline.